### ISSUE TITLE
--py3k does not catch outdated exception catch

### ISSUE THREAD

**Issue by hahahannes** [PyCQA/pylint#2582]
### Steps to reproduce
1. create file with
```python
try:
    pass
except Exception, e:
    pass
```
2. run pylint --py3k file
3. check output

### Current behavior
no output

### Expected behavior
message about outdated exception catch (Exception as e <-> Exception, e)

### pylint --version output
```
pylint 2.1.1
astroid 2.0.4
Python 3.7.0 (default, Aug 22 2018, 15:22:33)
[Clang 9.1.0 (clang-902.0.39.2)]
```

**Comment by PCManticore** [comment]
Yes, you are right, `pylint` doesn't catch this just yet.

**Comment by hahahannes** [comment]
Thanks for the fast answer! I can also create a PR but it was not clear to me how to I should handle the error message. Is there already an error code like E1603 + message for this case?

**Comment by Pierre-Sassoulas** [comment]
Won't do, we removed the python3 porting mode in 2.11.
